src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.c

```diff
@@ -1,3 +1,10 @@
+/*
+ * File: z_en_mushi2.c
+ * Overlay: ovl_En_Mushi2
+ * Description:
```

Comment:
This seems to be similar to z_en_insect.c in OoT, which uses this description:

```suggestion
 * Description: Bugs
```

Reply:
EnInsect seems to exist in MM too. Are you sure they're both bugs?

Reply:
I'm decently sure. Spawning this actor in kz definitely makes bugs show up:
![NKZJ01_2021-09-17_16-15-26](https://user-images.githubusercontent.com/12245827/133862831-cd4678c9-d557-4494-8564-4dcac0dd3cfd.png)

Reply:
To be more specific about their behavior, Insect (actor ID 0x0016) spawns a single bug that seemingly stays above ground forever, while Mushi2 (actor ID 0x017B) spawns three bugs that eventually burrow underground. So I guess you could more accurately describe this file as "burrowing bug".

Reply:
Hmm. I wonder if there are free-crawling bugs in MM but not OoT. I think there may be.

Reply:
I think you're right. `ACTOR_EN_INSECT` shows up in a variety of scenes, but Mushi2 never does. So you could call Insect "Free-standing bugs" and this just "Bugs" I suppose. The *exact* difference between them won't really be clear until Insect is decompiled, but Mushi2 is definitely just a bunch of bugs.